k8s/regions/frankfurt/dev.yaml

```diff
@@ -116,6 +116,7 @@ app:
     read_only: False
     redis_default_url: SECRET
     redis_helpfulvotes_url: SECRET
+    run_migrate: "true"
```

Comment:
Shouldn't this be false from Frankfurt?

Reply:
Not in dev. Dev is only in Frankfurt and is RW.